xonsh/dirstack.py

```diff
@@ -1,19 +1,20 @@
 # -*- coding: utf-8 -*-
 """Directory stack and associated utilities for the xonsh shell."""
-import os
-import glob
 import argparse
 import builtins
+import glob
+import os
 import subprocess
+import typing as tp
 
-from xonsh.lazyasd import lazyobject
-from xonsh.tools import get_sep
 from xonsh.events import events
+from xonsh.lazyasd import lazyobject
 from xonsh.platform import ON_WINDOWS
+from xonsh.tools import get_sep
```

Comment:
Please don't reorder imports in the future

Reply:
Ok noted.

Reply:
Thanks!